components/searchBar/SearchBar.module.scss

```diff
@@ -0,0 +1,20 @@
+.textBoxContainer {
+  display: flex;
+  border: solid 1px #bfc0c3;
+  max-width: 313px;
+  height: 40px;
+  border-radius: 8px;
+  background-color: white;
+  margin: 100px;
+  svg {
+    margin: 10px 16px;
+  }
+}
+
+.textBox {
+  border: none;
+  width: 100%;
+  border-radius: 8px;
```

Comment:
Why does the text nee border-radius?

Reply:
in the right side, the input element spread until the end of the container div, so its corners needed to be rounded.
now we have padding on the sides of the container so this line can be removed